ydb/services/ydb/backup_ut/ydb_backup_ut.cpp

```diff
@@ -472,24 +487,19 @@ void TestRestoreDirectory(const char* directory, TSchemeClient& client, TBackupF
 
 Y_UNIT_TEST_SUITE(BackupRestore) {
 
-    void Restore(NDump::TClient& client, const TFsPath& sourceFile, const TString& dbPath) {
-        auto result = client.Restore(sourceFile, dbPath);
-        UNIT_ASSERT_C(result.IsSuccess(), result.GetIssues().ToString());
-    }
-
-    auto CreateBackupLambda(const TDriver& driver, const TFsPath& pathToBackup, bool schemaOnly = false) {
-        return [&driver, &pathToBackup, schemaOnly](const char* table) {
-            Y_UNUSED(table);
-            // TO DO: implement NDump::TClient::Dump and call it instead of BackupFolder
-            NBackup::BackupFolder(driver, "/Root", ".", pathToBackup, {}, schemaOnly, false);
+    auto CreateBackupLambda(const TDriver& driver, const TFsPath& fsPath, const TString& dbPath = "/Root") {
+        return [&]{
```

Comment:
Здесь и далее:
```suggestion
        return [&] {
```

Или:
```cpp
        return [&]() {
```